source/eventcore/drivers/posix/select.d

```diff
@@ -79,6 +79,7 @@ final class SelectEventLoop : PosixEventLoop {
 
 	override void dispose()
 	{
+		super.dispose();
 	}
```

Comment:
In this case, just remove the override ? But it looks like this is no accident ?

Reply:
Yep, so that whoever messes with these next can't miss it.